parser/model/ddl.go

```diff
@@ -854,6 +854,19 @@ func (job *Job) IsRollbackable() bool {
 	return true
 }
 
+func (job *Job) NeedNotifyAndStopReorgWorker() bool {
```

Comment:
This function is complex(goes into more detail) and only used in ddl. I think putting it to `ddl/reorg.go` is better.

Reply:
This is a kind of design issue. 

The who procedure of this function is not out of Job, even though which is for reorg.go only.  Once we design the whole architecture as Object Oriented, we may do it like this, otherwise depends.